src/web/forms/fields/AutoCalc/__tests__/functions-test.js

```diff
@@ -42,3 +42,29 @@ describe('asha:bmi', () => {
       .toBeCloseTo(21.46);
   });
 });
+
+describe('asha:sum', () => {
+  const func = functions['asha:sum'];
+
+  it('calculates sum of arbitrary list of args', () => {
+    // Integers
+    expect(func(1, 2, 3)).toEqual(6);
+    expect(func(3.5, 1.5)).toEqual(5.0);
+    // const arg1 = 1;
+  });
+
+  it('adds together nothing if no arg/nullarg & turns into number (0)', () => {
```

Comment:
I believe 0 is more (mathematically) correct in terms of actual usage; 
adding nothing in mathematical and real-world usage context almost always means adding 0.
As the function of 'addition' implies that something should be a number, it may be more correct to expectation that the function always returns a number, and something is always summed...

Reply:
I see. It makes sense that sum of nothing (`sum( )`) is `0` in practical context.

But I don't think `sum(null)` is `0`.
`null` is not a number.
I think the function should return `null` or raise error in this case.

Reply:
You are correct and I fixed it so that null is handled appropriately.
'Null' is an object and should not be treated equal to 0 even though it is also falsy.